main.go

```diff
@@ -248,42 +249,52 @@ func Run(ctx context.Context) int {
 	}
 
 	g, ctx := errgroup.WithContext(ctx)
-	g.Go(func() error {
-		return admin.StartServer(ctx)
+	g.Go(func() (err error) {
+		if err = admin.StartServer(ctx); err != nil {
+			if !errors.Is(err, context.Canceled) {
+				pkgLogger.Errorf("Error starting admin server: %v", err)
```

Comment:
minor: should we prefer starting with a lowercase character for logs as we do with errors?

Reply:
We don't have a standard at the moment in this regard but I would say that it is not necessary. For errors it makes more sense IMO because we concatenate them so you might not want `error 1: Error 2` but want `error 1: error 2`. Either way no big deal, as long as we decide on one we can stick to it.